COMPILING.md

```diff
@@ -309,3 +309,22 @@ If compiling with cmake:
    ```
    cmake --build build
    ```
+
+## Use BDDs for guards
+
+There are two implementation for symex guards. The default one uses the
+internal representation of expression. The other one uses BDDs and
+though experimental, it is expected to have better performances on
+difficult programs, in particular when used in conjunction with CUDD.
```

Comment:
I don't know what a "difficult" program is - maybe just omit that? I think you've demonstrated performance improvements across the board for any runtime greater than a few seconds.